server/src/main/java/org/elasticsearch/action/support/TransportLocalClusterStateAction.java

```diff
@@ -61,4 +65,33 @@ protected final void doExecute(Task task, Request request, ActionListener<Respon
 
     protected abstract void localClusterStateOperation(Task task, Request request, ClusterState state, ActionListener<Response> listener)
         throws Exception;
+
+    // Remove the BWC support for the deprecated ?master_timeout parameter.
+    // NOTE: ensure each usage of this method has been deprecated for long enough to remove it.
+    @UpdateForV10(owner = UpdateForV10.Owner.DISTRIBUTED_COORDINATION)
```

Comment:
I marked Distributed as owner as I figured they own this class. I could also (additionally or instead) add the annotation to the REST action.